LibSWBF2.NET/Wrappers/Container.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading; 
+
+using LibSWBF2.Logging;
+
+
+namespace LibSWBF2.Wrappers
+{
+    public class Container : NativeWrapper
+    {
+        private static Dictionary<Type, uint> wrapperMap;
+        
+
+        public Container() : base(APIWrapper.Container_Initialize()){}
+        
+        static Container()
+        {
+            wrapperMap = new Dictionary<Type, uint>();
+            wrapperMap[typeof(Model)] = 0;
+        }        
+
+        public uint AddLevel(string path)
+        {
+            return APIWrapper.Container_AddLevel(NativeInstance, path);
+        }
+
+        public float GetProgress(uint handle)
+        {
+            return APIWrapper.Container_GetProgress(NativeInstance, handle);
+        }
+
+        public Level GetLevel(uint handle, bool block = false)
+        {
+            while (!IsDone())
+            {
+                if (block)
+                {
+                    Thread.Sleep(200);
+                } 
+                else 
+                {
+                    return null;
+                } 
+            }
+
+            return Level.FromNative(APIWrapper.Container_GetLevel(NativeInstance, handle));
+        }
```

Comment:
There's probably a better way besides ```Thread.Sleep```, but threading in C# is still very new to me...